brianlong/www.validators.app
# 184193382 Clean up old views

#### What's this PR do?
- Removes no longer used partials

#### How should this be manually tested?
- Go through validators & stake pools pages, click on the modals, confirm there're no errors related to missing partials

#### What are the relevant tickets?
- [URL to the PivotalTracker ticket](https://www.pivotaltracker.com/story/show/184193382)

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
